src/main/java/com/mraof/minestuck/item/ScalemateItem.java

```diff
@@ -0,0 +1,71 @@
+package com.mraof.minestuck.item;
+
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+import net.minecraft.util.ActionResult;
+import net.minecraft.util.ActionResultType;
+import net.minecraft.util.Hand;
+import net.minecraft.util.SoundEvents;
+import net.minecraft.world.World;
+import net.minecraftforge.common.Tags;
+
+public class ScalemateItem extends Item
+{
+    private Item newItem;
```

Comment:
This seems like it's only used as a local variable, and should probably be one too